### PR TITLE
Rename cloud part and use cobra cmd

### PR DESCRIPTION
**Pull request by fisherxu**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:
>
> /kind api-change
> /kind bug

/kind cleanup
> /kind design
> /kind documentation
> /kind test
> /kind failing-test
> /kind feature

Xref: #980 

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: cloud/cmd/cloudcore/app/server.go [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+	"github.com/kubeedge/beehive/pkg/core"
+	"github.com/kubeedge/kubeedge/cloud/pkg/cloudhub"
+	"github.com/kubeedge/kubeedge/cloud/pkg/devicecontroller"
+	"github.com/kubeedge/kubeedge/cloud/pkg/edgecontroller"
+)
+
+func NewCloudCoreCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use: "cloudcore",
+		Long: `Cloud Core is the core cloud part of KubeEdge, which contains three modules: cloudhub,
```
Comment (kevin-wangzefeng) [inline cloud/cmd/cloudcore/app/server.go]: s/Cloud Core/CloudCore/g

File: docs/proposals/keadm-scope.md [hunk 2]
```diff
@@ -183,7 +183,7 @@ keadm join --edgecontrollerip=10.20.30.40 --edgenodeid=testing123 --kubeedge-ver
 
 Flags:
       --docker-version string[="18.06.0"]          Use this key to download and use the required Docker version (default "18.06.0")
-  -e, --edgecontrollerip string                    IP address of KubeEdge edgecontroller
+  -e, --cloudcoreip string                         IP address of KubeEdge cloud core
```
Comment (kevin-wangzefeng) [inline docs/proposals/keadm-scope.md]: s/Cloud Core/CloudCore/

File: keadm/app/cmd/edge/join.go [hunk 3]
```diff
@@ -98,9 +98,9 @@ func addJoinOtherFlags(cmd *cobra.Command, joinOptions *types.JoinOptions) {
 	cmd.Flags().StringVarP(&joinOptions.K8SAPIServerIPPort, types.K8SAPIServerIPPort, "k", joinOptions.K8SAPIServerIPPort,
 		"IP:Port address of K8S API-Server")
 
-	cmd.Flags().StringVarP(&joinOptions.EdgeControllerIP, types.EdgeControllerIP, "e", joinOptions.EdgeControllerIP,
-		"IP address of KubeEdge edgecontroller")
-	cmd.MarkFlagRequired(types.EdgeControllerIP)
+	cmd.Flags().StringVarP(&joinOptions.CloudCoreIP, types.CloudCoreIP, "e", joinOptions.CloudCoreIP,
+		"IP address of KubeEdge Cloud Core")
```
Comment (kevin-wangzefeng) [inline keadm/app/cmd/edge/join.go]: s/Cloud Core/CloudCore/

File: docs/setup/installer_setup.md [hunk 4]
```diff
@@ -162,7 +162,7 @@ Flags:
 
 **NOTE:** All the below steps are executed as root user, to execute as sudo user ,Please add **sudo** infront of all the commands
 
-### 1. Deploy KubeEdge edgeController (With K8S Cluster)
+### 1. Deploy KubeEdge cloudCore (With K8S Cluster)
```
Comment (kevin-wangzefeng) [inline docs/setup/installer_setup.md]: s/cloudCore/CloudCore/
s/K8S/K8s/g

File: cloud/cmd/cloudcore/app/server.go (hunk 1 above)
Comment (fisherxu) [reply cloud/cmd/cloudcore/app/server.go]: Done.

**Comment by fisherxu** [comment]
PR ready to review now.
/cc @kevin-wangzefeng @sids-b @subpathdev @chendave @rohitsardesai83 @kadisi   Ptal :)

**Comment by kadisi** [comment]
/lgtm 
 this is a big good change

**Pull request merged**